distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3985,7 +3986,7 @@ None
 
 Using the REST Endpoint for example, request a metacard with the transform option set to the HTML shortname.
 ----
-http://localhost:8181/services/catalog/0123456789abcdef0123456789abcdef?transform=html
+${public_url}/services/catalog/0123456789abcdef0123456789abcdef?transform=html
```

Comment:
I would say yes, just to encourage using the secure_url as a default. We could, if we think it's necessary, include a note that calls out that you _can_ substitute the public_url if that's what you're using. I don't know if that is overexplaining, though.